tests/baselines/reference/exportNestedNamespaces.types

```diff
@@ -7,13 +7,13 @@ exports.n = {};
 >{} : typeof __object
 
 exports.n.K = function () {
->exports.n.K = function () {    this.x = 10;} : () => void
->exports.n.K : () => void
+>exports.n.K = function () {    this.x = 10;} : typeof (Anonymous function)

```

Comment:
Rather than `(Anonymous function)`, can we wire this up so it has name `K`?

Reply:
Yep, I'm working on it. I'm trying to understand the machinery that decides between `K`, `n.K` and `exports.n.K`.